### PR TITLE
Internal dev README (potentially user-facing README)

### PR DESCRIPTION
**Pull request by dbogunowicz**
A simple guide to using the new sparsezoo.

I would like this document to end up as a part of our user-facing docs. Let's discuss what properties this would need to have to be as useful as possible for our users.

### CODE REVIEW

File: README_.md
Comment (corey-nm) [inline README_.md:49]:
```suggestion
# method 2
```

**Review by corey-nm — DISMISSED** [review]
Nice document!

**Comment by corey-nm** [comment]
@dbogunowicz what's the plan with this doc? still want to merge or no?

**Comment by dbogunowicz** [comment]
@corey-nm absolutely. It ended up in limbo, waiting for reviews. Let me ping the rest of the team to decide how to complete this PR.

**Review by bfineran — DISMISSED** [review]
LGTM overall. Please re verify all code examples before landing since it's been a few months

**Review by KSGulin — DISMISSED** [review]

File: README.md
Comment (dbogunowicz) [inline README.md:86]:
```suggestion
The SparseZoo Python API enables you to search and download sparsified models. Code examples are given below.
```

File: README.md
Comment (dbogunowicz) [inline README.md:87]:
```suggestion
We encourage the users to load SparseZoo models by copying a stub directly from a [model page]((https://sparsezoo.neuralmagic.com/)).
```

File: README.md
Comment (dbogunowicz) [inline README.md:130]:
```suggestion
```

File: README.md
Comment (dbogunowicz) [inline README.md:133]:
```suggestion
Once the model is initialized from a stub, it may be downloaded either by calling the `download()` method or by invoking a `path` property. Both pathways are universal for all the files in SparseZoo. Invoking the `path` property will always trigger file download unless the file has already been downloaded.
```

File: README.md
Comment (dbogunowicz) [inline README.md:342]:
```suggestion
```

**Comment by dbogunowicz** [comment]
Revisited the PR, fused my write-up with the repositories README.md, added some missing features to the documentation, and tested all the pathways.

**Review by corey-nm — DISMISSED** [review]
🎉

File: README.md
Comment (jeanniefinks) [inline README.md:87]:
```suggestion
We encourage users to load SparseZoo models by copying a stub directly from a [model page]((https://sparsezoo.neuralmagic.com/)).
```

File: README.md
Comment (jeanniefinks) [inline README.md:341]: Is the end of the next paragraph missing some words or was this phrase left in? e.g., 
Default `~/.cache/sparsezoo/`

**Review by jeanniefinks — DISMISSED** [review]
Reads great! Minor minor comments!

File: README.md
Comment (dbogunowicz) [reply README.md:341]: Nope, it is just an empty line.

File: README.md
Comment (dbogunowicz) [inline README.md:341]:
```suggestion
```

**Review by corey-nm — APPROVED** [review]

**Review by markurtz — APPROVED** [review]
@dbogunowicz  can we get this cherry picked over to 1.2 as well?

**Comment by dbogunowicz** [comment]
@markurtz for sure

**Pull request merged**
